server/main_server/src/rodsServer.cpp

```diff
@@ -454,6 +455,12 @@ namespace
         }
     } // log_stacktrace_files
 
+
+    std::atomic<bool> will_reload_config = false;
```

Comment:
Move this closer to the beginning of the file so that all globals are grouped in a general location.

Reply:
`will_reload_config` -> `reload_server_config`